platform/viewer/public/config/multiple.js

```diff
@@ -9,7 +9,7 @@ window.config = {
     },
   ],
   extensions: [],
-  modes: [],
+  modes: ['@ohif/mode-test'],
```

Comment:
Run in mode-test for the multiple back end so it looks like it does during testing.